### PR TITLE
feat(ONEPLAT-255): opc-header component

### PR DESCRIPTION
**Pull request by diwanshi**
# Closes/Fixes/Resolves
https://projects.engineering.redhat.com/browse/ONEPLAT-255

### 

# Explain the feature/fix
A standardized web component based on Lit Element for Red Hat One Platform Header that uses Patternfly Breadcrumb and Links as button HTML component

## Does this PR introduce a breaking change
No

## Screenshots

<details>
<summary>View Screenshots</summary>

Screenshot_2020-09-03 opc-header

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code
- [x] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
#### If it is a new component
- [x] Does your component follow One Platform style guidelines?
- [x] Does your component web accessibility standards? [Helper Doc](https://www.w3.org/TR/wai-aria-1.1/)
- [x] Does your component support desktop screen sizes (350px, 720px, 1150px ,1920px)
#### Browsers you have tested in
- [ ] Latest two versions of Mozilla Firefox and Google Chrome supported by Red Hat Enterprise Linux distribution
- [ ] Google Chrome [Latest 2 versions]
- [ ] Mozilla Firefox [Latest 2 versions]
- [ ] Microsoft Edge [Latest 2 versions]
- [ ] Apple Safari [Latest 2 versions]
- [ ] Microsoft Internet Explorer 11

### CODE REVIEW

File: packages/opc-header/src/opc-header.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
+@import "@one-platform/opc-styles";
+
+$LOCAL: opc-header;
+
+$variables: (
+  //Add your variables here
+);
```
Comment (hybridx) [inline packages/opc-header/src/opc-header.scss:7]: @diwanshi should we remove this because we are using :host
```suggestion
```

**Review by hybridx — APPROVED** [review]
LGTM

File: packages/opc-header/README.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
+# opc-header component 👋
+
+![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
+
+A standardized web component based on Lit Element for Red Hat One Platform Header that uses Patternfly Breadcrumb and Links as button HTML component
```
Comment (hybridx) [inline packages/opc-header/README.md:7]: We might need to add patternfly unpkg link for reference

**Review by hybridx — CHANGES_REQUESTED** [review]

File: packages/opc-header/README.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+```html
+ <opc-header title="OPC Header" theme="dark">
+  <opc-header-breadcrumb slot="breadcrumb"></opc-header-breadcrumb>
+  <opc-header-links slot="links"></opc-header-links>
+ </opc-header>
+```
+
+Header with lightDOM
+```html
+ <opc-header title="OPC Header" theme="red">
```
Comment (gisumit) [inline packages/opc-header/README.md:70]: the **title** is a native global attribute for an element... can we change it another name

**Review by gisumit — CHANGES_REQUESTED** [review]

File: packages/opc-header/README.md (hunk 3 above)
Comment (diwanshi) [reply packages/opc-header/README.md:72]: @gisumit yes makes sense

File: packages/opc-header/src/opc-header.scss (hunk 1 above)
Comment (diwanshi) [reply packages/opc-header/src/opc-header.scss:7]: umm we are gonna need this soon as we work on variable naming automation. do u think we should still remove it?

**Review by hybridx — APPROVED** [review]

**Comment by hybridx** [comment]
Screenshot 2020-09-04 at 5 57 21 PM
@diwanshi @gisumit Angular application testing complete

File: packages/opc-header/src/opc-header.scss (hunk 1 above)
Comment (hybridx) [reply packages/opc-header/src/opc-header.scss:7]: Ohh okay, cause there is a function in opc-styles known as opc-get-var(). @gisumit Do we have examples?

**Review by sayak-sarkar — APPROVED** [review]

**Review by ghanlohar — APPROVED** [review]
Looks good.

**Pull request merged**